src/api/services/notifications/tasks.py

```diff
@@ -47,37 +56,52 @@ def send_email(**kwargs):
     :kwarg recipients:
     :return bool:
     """
-    if not kwargs["sender"] or not kwargs["sender"].strip():
-        raise ValueError("sender address missing")
-
-    validate_email(kwargs["recipients"])
-
-    if os.getenv("MAIL_GUN_TEST"):
-        return True
-
-    mail_gun_url = flask_app.config["MAIL_GUN_URL"]
-    mail_gun_api_key = flask_app.config["MAIL_GUN_API_KEY"]
-
-    data = {
-        "from": kwargs["sender"],
-        "to": kwargs["recipients"],
-        "subject": kwargs["subject"],
-        "text": kwargs["message"],
-    }
-    if kwargs.get("html"):
-        data.update(dict(html=kwargs.get("html")))
-    response = requests.post(
-        mail_gun_url, auth=("api", mail_gun_api_key), data=data
-    )
-    if response.status_code == 200:
-        return True
-    else:
-        return False
+    with app.app_context():
+        if not kwargs.get("sender") or not kwargs.get("sender").strip():
+            raise ValueError("sender address missing")
+
+        validate_email(kwargs["recipients"])
+
+        if current_app.config.get("MAIL_GUN_TEST"):
+                return True
+
+        msg = Message(subject=kwargs["subject"],
+                      recipients=kwargs["recipients"], sender=kwargs["sender"])
+        msg.body = kwargs["message"]
+
+        if kwargs.get("html"):
+            msg.html = kwargs.get("html")
+
+        mail_sender.send(msg)
+
+        # NOTE:  These will be used in future when the problems with MailGun
```

Comment:
Git can do a much better job than, leaving comments. You can safely delete this if required `git checkout <previous version>` will suffice to bring it back.